### ISSUE TITLE
Graphic glitches with stream output support on GCN 1.0

### ISSUE THREAD

**Issue by ahmed-elsayed2017** [doitsujin/dxvk#719]
### Software information
The Witcher 3 - Medium settings - 720P resolution

### System information
- GPU: AMD HD 8750M
- Driver: Mesa 18.3 with the RADV patches for VK_EXT_transform_feedback
https://aur.archlinux.org/packages/mesa-transform-feedback-git
- Wine version: esync-staging-pba-3.18
- DXVK version:  0.90

I couldn't see this monster before (Ekhidna), but now it is appearing with graphic glitches.

![screenshot_20181021_024154](https://user-images.githubusercontent.com/29783425/47262020-9f947380-d4dc-11e8-82ce-5ab4c7ff3bd8.png)

### Log files
[witcher3_d3d11.log](https://github.com/doitsujin/dxvk/files/2498685/witcher3_d3d11.log)
[witcher3_dxgi.log](https://github.com/doitsujin/dxvk/files/2498686/witcher3_dxgi.log)

**Comment by ahmed-elsayed2017** [comment]
https://github.com/doitsujin/dxvk/issues/695

**Comment by doitsujin** [comment]
Works fine on my RX 480, must be specific to your GPU.

**Comment by ahmed-elsayed2017** [comment]
Is it a new issue or is it already known?

**Comment by ahmed-elsayed2017** [comment]
Do I need to report about this bug to Mesa?

**Comment by ahmed-elsayed2017** [comment]
Thanks. They will fix it soon.

https://bugs.freedesktop.org/show_bug.cgi?id=108508#

**Comment by chelobaka** [comment]
RADV support of GCN 1.0 HW is in poor state itself. I tried Batman: Arkham Origins on HD7750 1GB card: the performance was suboptimal, after some time I started to get missing textures and finally the game hang. Natively this game runs fine on this HW.

**Comment by ahmed-elsayed2017** [comment]
I didn't try Batman Arkham Origins on Linux, but I played Batman Arkham Knight, and it works without any issues.

**Comment by chelobaka** [comment]
It was Mesa 18.1.x and LLVM 6.0 at my side so maybe it's OK nowadays.

**Comment by ahmed-elsayed2017** [comment]
When I use Wine without DXVK the objects render correctly but the textures is darker than it should be. Could you explain that, please? @doitsujin 

![screenshot_20181101_021129](https://user-images.githubusercontent.com/29783425/47856832-d168dc80-ddf0-11e8-8902-e2d2b1c66432.png)

**Comment by doitsujin** [comment]
No, I actually can't, since both DXVK and wined3d are rendering Witcher 3 correctly on my machine.

**Comment by ahmed-elsayed2017** [comment]
@doitsujin Guess what?!

I updated the kernel to v. 4.19 and The Witcher 3 is working perfectly now.
